lib/markdown.ts

```diff
@@ -0,0 +1,15 @@
+import { remark } from 'remark';
+import rehypeSlug from 'rehype-slug';
+import remarkRehype from 'remark-rehype';
+import rehypeStringify from 'rehype-stringify';
+
+export default async function markdownToHtml(markdown: string) {
+  const result = await remark()
+    .use(remarkRehype, { allowDangerousHtml: true })
+    .use(rehypeSlug)
+    // .use(rehypeAutolinkHeadings)
+    .use(rehypeStringify, { allowDangerousHtml: true })
+    .process(markdown);
+
+  return result.toString();
+}
```

Comment:
What do you think about using [mdx](https://mdxjs.com/) instead  ***remark***?

If we use mdx then we can have react components inside the blog post.

Reply:
I have created a pr with mdx integration:  https://github.com/gironajs/gironajs.com/pull/4

Reply:
Sounds good to use MDX 👌